antora.yml

```
@@ -51,30 +54,34 @@ asciidoc:
     jobs_service_image_postgresql_name: logic-jobs-service-postgresql-rhel8
     jobs_service_image_postgresql: registry.redhat.io/openshift-serverless-1/logic-jobs-service-postgresql-rhel8
     jobs_service_image_postgresql_url: https://catalog.redhat.com/software/containers/openshift-serverless-1/logic-jobs-service-postgresql-rhel8/6614eddbaeb155f6aae45385
-    jobs_service_image_usage_url: https://github.com/kiegroup/kogito-images/tree/9.100.x-prod#jobs-services-all-in-one
 
     #
     #  Versions 
     #
-    quarkus_version: 3.8.4.redhat-00002
+    quarkus_version: 3.8.6.redhat-00004
     quarkus_platform_version: 3.8.4.redhat-00002
```

don't we need to update quarkus platform version?


Prod version is:
```suggestion
    quarkus_platform_version: 3.8.6.SP2-redhat-00002
```